### ISSUE TITLE
TypeError on get_blog()

### ISSUE THREAD

**Issue by anthonyadavisii** [holgern/beem#146]
#### Project Information

* Repository: https://github.com/holgern/beem

* Project Name: Beem

* Publisher (if applicable): @holger80

#### Expected behavior

The following line would obtain the account's last main post.

```
post = sfr.get_blog(limit=1)[0]
```
Which is used by the @steemflagrewards to implement our posting frequency limit (1 report / 4 hours)

![image.png](https://ipfs.busy.org/ipfs/QmUZPAty4tycKGLYfbdJRRmcC8z7RfHJTVxXUMcxG4VAKn)

It has been working up until earlier today. I've successfully ensured the library was up to date but issue persists.

#### Actual behavior

![image.png](https://ipfs.busy.org/ipfs/QmfN4qSYG16xCCZ5EwtCz98LeG2w3dCuNh2cpNTxad9SRg)

#### How to reproduce

You may reproduce the bug interactively as follows:

```
from beem import Steem
from beem.account import Account
from beem.comment import Comment
stm = Steem(nodes='<insert node>')
sfr = Account('steemflagrewards', steem_instance=stm)
post = sfr.get_blog(limit=1)[0]
```
Which results in error as follows:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/sfr/miniconda3/lib/python3.6/site-packages/beem/account.py", line 738, in get_blog
    Comment(c["comment"], steem_instance=self.steem) for c in self.steem.rpc.get_blog({'account': account, 'start_entry_id': start_entry_id, 'limit': limit}, api='follow')["blog"]
TypeError: list indices must be integers or slices, not str

* App version: Beem version 0.20.16

* Operating system: Ubuntu 18.04.1 LTS

#### Recording Of The Bug

![EF08F3C5-EB8F-4BD2-A4AE-038D96E282F0.GIF](https://ipfs.busy.org/ipfs/QmV3bnAsMCXeRAPsVTsJRg4GNKKTvgCHUzMBVfUyrQpjzf)

**Comment by fel1xw** [comment]
I have the same issue, and the same problem of type `TypeError: list indices must be integers or slices, not str` also occurs for `account.get_followers()`.

**Comment by holgern** [comment]
Fixed in 0267cf1f0eb14a14d62e11968c3eb4ee24d476e8
